zos_concepts/gdg_datasets/create_copy_submit/README.md

```diff
@@ -0,0 +1,87 @@
+# Create GDG, copy delete and submit of Generation Data Group
```

Comment:
My suggestion would be to stay as close to the existing naming of non-GDG use cases so they can see the difference, either the older one use case be renamed or this one change (preferably) 

What exists today is:
`Copy Directory to PDS, Edit member and Submit Job`

How about:
`Using a GDG, copy Directory to GDS and Submit Job`

If you plan to edit, then:
`Using a GDG, copy Directory to GDS, Edit member and Submit Job`